src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -64,3 +66,12 @@ def post(self, request: Request, organization) -> Response:
             onboarding_tasks.try_mark_onboarding_complete(organization.id)
 
         return Response(status=204)
+
+    def get(self, request: Request, organization) -> Response:
+
+        tasks_to_serialize = list(
+            onboarding_tasks.fetch_onboarding_tasks(organization, request.user)
+        )
```

Comment:
Does this actually need to be wrapped with `list`? Is it a generator, can that just be passed directly into the serializer?

Reply:
I copied it from here https://github.com/getsentry/sentry/blob/fc95869924926d555bdf3208a786fded56a28821/src/sentry/api/serializers/models/organization.py#L376-L377 so probably yes

Reply:
probably fine

Reply:
I get an error if I remove the list